test/standalone/test-trace-writer.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var assert = require('assert');
+var nock = require('nock');
+var cls = require('../../lib/cls.js');
+var agent = require('../..');
+var request = require('request');
+
+nock.disableNetConnect();
+
+var uri = 'https://cloudtrace.googleapis.com';
+var path = '/v1/projects/0/traces';
```

Comment:
Yup, that was a last minute change to require the environment set. It has been fixed.